velox/docs/functions/spark/array.rst

```diff
@@ -0,0 +1,43 @@
+=============================
+Array Functions
+=============================
+
+.. function:: size(x) -> bigint
+
+    Returns the size of the array ``x``. The function returns null for null input
+    if spark.sql.legacy.sizeOfNull is set to false or spark.sql.ansi.enabled is set to true.
+    Otherwise, the function returns -1 for null input. With the default settings,
+    the function returns -1 for null input.
+
+.. function:: in(x, element) -> boolean
+
+    Returns true if the array ``x`` equals to any value of ``element``.
+
+.. function:: array_sort(array(E)) -> array(E)
+
+     Returns an array which has the sorted order of the input array x. The elements of x must
+     be orderable. Null elements will be placed at the end of the returned array.::
+
+        SELECT array_sort(ARRAY [1, 2, 3]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [3, 2, 1]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [2, 1, NULL]; -- [1, 2, NULL]
+        SELECT array_sort(ARRAY [NULL, 1, NULL]); -- [1, NULL, NULL]
+        SELECT array_sort(ARRAY [NULL, 2, 1]); -- [1, 2, NULL]
+
+.. function:: sort_array(array(E)[, ascendingOrder]) -> array(E)
+
+     Returns an array which has the sorted order of the input array x. The elements of x must
+     be orderable. Null elements will be placed at the beginning of the returned array in ascending
+     order or at the end of the returned array in descending order.::
+
+        SELECT sort_array(ARRAY [1, 2, 3]); -- [1, 2, 3]
+        SELECT sort_array(ARRAY [3, 2, 1], true); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [2, 1, NULL, true]; -- [NULL, 1, 2]
+        SELECT array_sort(ARRAY [NULL, 1, NULL], false); -- [1, NULL, NULL]
+        SELECT array_sort(ARRAY [NULL, 2, 1]); -- [null, 1, 2]
+
+.. function:: getarrayitem(array(E), index) -> E
```

Comment:
For some reason, I do not see this function in https://spark.apache.org/docs/2.4.1/api/sql/index.html Is this a valid Spark SQL function?

Reply:
just removed it.